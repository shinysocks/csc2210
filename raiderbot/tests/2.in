+--------------------+
|     #              |
|  **          -     |
| #         #        |
|                    |
| O                  |
|    *#      +   #   |
|                    |
|  ####              |
|              ##    |
|                    |
+--------------------+
4 5
seesw

