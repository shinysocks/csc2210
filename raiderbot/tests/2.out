Initial state: Robot at 4, 5 (1 gold)
+--------------------+
|     #              |
|  **          -     |
| #         #        |
|                    |
| O                  |
|    R#      +   #   |
|                    |
|  ####              |
|              ##    |
|                    |
+--------------------+
========================================
Executing seesw

Error: robot with current status of Robot at 6, 7 (1 gold) broke while executing seesw
+--------------------+
|     #              |
|  **          -     |
| #         #        |
|                    |
| O                  |
|     #      +   #   |
|                    |
|  ####R             |
|              ##    |
|                    |
+--------------------+
========================================
